src/System.Windows.Forms/src/System/Windows/Forms/KeysConverter.cs

```diff
@@ -156,68 +165,47 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 else
                 {
                     // Parse an array of key tokens.
-                    //
                     string[] tokens = text.Split(new char[] { '+' });
                     for (int i = 0; i < tokens.Length; i++)
                     {
                         tokens[i] = tokens[i].Trim();
                     }
 
                     // Now lookup each key token in our key hashtable.
-                    //
                     Keys key = (Keys)0;
                     bool foundKeyCode = false;
 
                     for (int i = 0; i < tokens.Length; i++)
                     {
                         object obj = KeyNames[tokens[i]];
```

Comment:
There's no need to box the value
```suggestion
                        Keys currentKey = KeyNames[tokens[i]];
```